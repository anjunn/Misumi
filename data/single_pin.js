module.exports = {
	quotationCondition: {
    quantity :"2",
    material :"SKH51 (58～60HRC)",
    surfaceTreatment :"なし",
    ToleranceGrade :"並級",
  },
  quotionConditionInPartsView: {
    quantity :"2",
  },
  thankyou: {
    heading:"注文手続き",
  }
};