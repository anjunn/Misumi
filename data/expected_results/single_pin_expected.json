{

  "projectName": "single_pin.x_t",

  "quotationCondition": {
    "material": "SKH51 (58～60HRC)",
    "surfaceTreatment": "なし",
    "ToleranceGrade": "並級"
  },

  "quotationConditionInPartsView": {
    "quantity": 2
  },

  "partNames": {
    "part1": "コアピン"
  }

}